sdk/template/azure-sdk-template/README-EXAMPLE.md

```diff
@@ -0,0 +1,125 @@
+# Azure Template client library for Java
+
+This is a template README for Azure Java libraries, this is used as a template for new libraries added into this repository.
+
+[Source code][source_code] | [Package (Maven)][package] | [API reference documentation][api_documentation]| [Product documentation][search_docs] | [Samples][samples]
+
+## Getting started
+
+### Include the package
+
+If your module is include in [azure-sdk-bom](azure_sdk_bom) use the BOM depencency management include.
```

Comment:
@JonathanGiles is this what you want to see being used moving forward with the BOM in play.